{
 "PRed": [
  {
   "PR": {
    "labels": [],
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.16.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.17.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1587031614.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python312"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 4,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.18.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 5,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.19.0"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 6,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "0.19.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": {
  "exception": "Error running check-solvable in container - return code 137:\ncmd: ['docker',\n 'run',\n '-e',\n 'CF_TICK_IN_CONTAINER=true',\n '--security-opt=no-new-privileges',\n '--read-only',\n '--cap-drop=all',\n '--mount',\n 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=6000000000',\n '-m',\n '6000m',\n '--cpus',\n '1',\n '--ulimit',\n 'nofile=1024:1024',\n '--ulimit',\n 'nproc=2048:2048',\n '--rm',\n '-i',\n '--mount',\n 'type=bind,source=/tmp/tmp0m4lv2ha/locan-feedstock,destination=/cf_tick_dir,readonly',\n 'ghcr.io/regro/conda-forge-tick:master',\n '/opt/conda/envs/cf-scripts/bin/python',\n '-u',\n '/opt/autotick-bot/docker/run_bot_task.py',\n 'check-solvable',\n '--timeout',\n '600',\n '--verbosity',\n '2',\n '--log-level',\n 'info']\noutput: ''",
  "traceback": [
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 567, in _run_migrator_on_feedstock_branch",
   "    migrator_uid, pr_json = run(",
   "                            ^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 360, in run",
   "    solvable, errors, _ = is_recipe_solvable(",
   "                          ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/solver_checks.py\", line 79, in is_recipe_solvable",
   "    return _is_recipe_solvable_containerized(",
   "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/solver_checks.py\", line 138, in _is_recipe_solvable_containerized",
   "    data = run_container_task(",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/utils.py\", line 244, in run_container_task",
   "    raise ContainerRuntimeError(",
   "conda_forge_tick.utils.ContainerRuntimeError: Error running check-solvable in container - return code 137:",
   "cmd: ['docker',",
   " 'run',",
   " '-e',",
   " 'CF_TICK_IN_CONTAINER=true',",
   " '--security-opt=no-new-privileges',",
   " '--read-only',",
   " '--cap-drop=all',",
   " '--mount',",
   " 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=6000000000',",
   " '-m',",
   " '6000m',",
   " '--cpus',",
   " '1',",
   " '--ulimit',",
   " 'nofile=1024:1024',",
   " '--ulimit',",
   " 'nproc=2048:2048',",
   " '--rm',",
   " '-i',",
   " '--mount',",
   " 'type=bind,source=/tmp/tmp0m4lv2ha/locan-feedstock,destination=/cf_tick_dir,readonly',",
   " 'ghcr.io/regro/conda-forge-tick:master',",
   " '/opt/conda/envs/cf-scripts/bin/python',",
   " '-u',",
   " '/opt/autotick-bot/docker/run_bot_task.py',",
   " 'check-solvable',",
   " '--timeout',",
   " '600',",
   " '--verbosity',",
   " '2',",
   " '--log-level',",
   " 'info']",
   "output: ''",
   ""
  ]
 },
 "pinning_version": "2024.03.13.18.01.29",
 "pre_pr_migrator_attempts": {
  "pypy38": 65
 },
 "pre_pr_migrator_status": {
  "pypy38": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/10038685267\">bot CI job</a>): main:\nError running check-solvable in container - return code 137:\ncmd: ['docker',\n 'run',\n '-e',\n 'CF_TICK_IN_CONTAINER=true',\n '--security-opt=no-new-privileges',\n '--read-only',\n '--cap-drop=all',\n '--mount',\n 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=6000000000',\n '-m',\n '6000m',\n '--cpus',\n '1',\n '--ulimit',\n 'nofile=1024:1024',\n '--ulimit',\n 'nproc=2048:2048',\n '--rm',\n '-i',\n '--mount',\n 'type=bind,source=/tmp/tmp0m4lv2ha/locan-feedstock,destination=/cf_tick_dir,readonly',\n 'ghcr.io/regro/conda-forge-tick:master',\n '/opt/conda/envs/cf-scripts/bin/python',\n '-u',\n '/opt/autotick-bot/docker/run_bot_task.py',\n 'check-solvable',\n '--timeout',\n '600',\n '--verbosity',\n '2',\n '--log-level',\n 'info']\noutput: ''"
 },
 "smithy_version": "3.31.1"
}